### ISSUE TITLE
paddle.fluid.nets.img_conv_group()的参数里没有卷积步长

### ISSUE THREAD

**Issue by liguoyu666** [PaddlePaddle/Paddle#27898]
paddle.fluid.nets.img_conv_group(input, conv_num_filter, pool_size, conv_padding=1, conv_filter_size=3, conv_act=None, param_attr=None, conv_with_batchnorm=False, conv_batchnorm_drop_rate=0.0, pool_stride=1, pool_type='max', use_cudnn=True)

该组合操作的参数里没有找到conv_stride，请问是怎么计算的，默认为１吗？
conv_num_filter是卷积后的通道数吗？卷积后是32个通道必须写成 (32,32)？

**Comment by wangchaochaohu** [comment]
stride 默认是1 conv_num_filter 是卷积之后每个group的通道 如果是一个group 的话 想要卷积之后 输出channel 数目是32 只用conv_num_filter 等于32 就行了 这个OP是一个组合网络 conv + batchnorm + pool

**Comment by liguoyu666** [comment]
> stride 默认是1 conv_num_filter 是卷积之后每个group的通道 如果是一个group 的话 想要卷积之后 输出channel 数目是32 只用conv_num_filter 等于32 就行了 这个OP是一个组合网络 conv + batchnorm + pool

谢谢解答，还有点疑问：
1. 这个组合网络的的卷积步长是只能为1吗？没看到修改的参数。
2. 我要用步长为2的卷积+BN，不要pool层，能否用该组合网络？参数里面pool貌似必须设定参数才行。
3. 有没有类似pytorch里的torch.nn.Sequential()，连续定义多个卷积、BN、pool等操作的API。
3. conv_num_filter （list | tuple） - 卷积中使用的滤波器数。若要求卷积后的通道为32，这个参数貌似不能设为32，而要（32, 32）才行。分成的group怎么设置的，没看到group的参数。

**Comment by wangchaochaohu** [comment]
1 组合卷积步长只能是1 
2 如果不想要pool ，但是想要使用这个API可以使用一种特殊的pool pool_size 等于1 的话是对中间输出结果并不会改变
3. Paddle中最新2.0 已经支持 Sequential(）  具体可以参考 https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/tutorial/quick_start/high_level_api/high_level_api.html#sequential

4  这个API已经不建议使用 这个group 对每个conv_num_filter中元素默认是1

**Comment by liguoyu666** [comment]
> 1 组合卷积步长只能是1
> 2 如果不想要pool ，但是想要使用这个API可以使用一种特殊的pool pool_size 等于1 的话是对中间输出结果并不会改变
> 3. Paddle中最新2.0 已经支持 Sequential(） 具体可以参考 https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/tutorial/quick_start/high_level_api/high_level_api.html#sequential
> 
> 4 这个API已经不建议使用 这个group 对每个conv_num_filter中元素默认是1

谢谢
